spacy_streamlit/visualizer.py

```diff
@@ -171,6 +171,7 @@ def visualize_ner(
     colors: Dict[str, str] = {},
     key: Optional[str] = None,
     manual: Optional[bool] = False,
+    extra_options: Optional[Dict] = {}
```

Comment:
```suggestion
    extra_options: Optional[Dict] = {},
```
(cf formatting with `black`)